src/middleware/log.ts

```diff
@@ -9,9 +9,28 @@ export const logMiddleware = (
 ) => {
   if (environment === "dev") {
     res.on("finish", () => {
+      const statusCode = res.statusCode;
+      const statusHighlight =
+        statusCode >= 400 ? `[ERROR ${statusCode}]` : `[Status: ${statusCode}]`;
+
+      const logBody =
+        statusCode >= 400
+          ? {
+              url: req.originalUrl,
```

Comment:
This all gets handled by the new `pino-http` middleware. All logs using `req.log` will include this metadata (and more).